.github/workflows/main.yml

```
@@ -53,6 +53,9 @@ jobs:
       if: matrix.skip-wsl-only
       run: |
         rm -r src/Examples/DAE/petsc*.ipynb
+    - name: Skip notebooks that require Alamo
+      run: |
+        rm -r src/Examples/SurrMod/FlowsheetOptimization/ALAMO_flowsheet_optimization.ipynb
```

The `-r` flag is only needed to remove directories, and is not needed when the argument is a file (or multiple files).

```suggestion
        rm src/Examples/SurrMod/FlowsheetOptimization/ALAMO_flowsheet_optimization.ipynb
```